istioctl/cmd/auth.go

```diff
@@ -158,4 +223,8 @@ func init() {
 		"Show additional information (e.g. SNI and ALPN)")
 	checkCmd.PersistentFlags().StringVarP(&configDumpFile, "file", "f", "",
 		"Check the TLS/JWT/RBAC setting from the config dump file")
+	upgradeCmd.PersistentFlags().StringVarP(&v1PolicyFile, "file", "f", "",
+		"Authorization policy file")
+	upgradeCmd.PersistentFlags().StringSliceVarP(&serviceSpecs, "service", "s", []string{},
+		"Service specification that provides the mapping relationship between service name and workload labels")
```

Comment:
Maybe mention the input is a yaml file with service spec. When I first read it, I thought it was something like `service-name: workload-label` by the "provides the mapping relationship between service name and workload labels"

Reply:
Updated the comment to make it clear it's k8s service resource.